.github/workflows/test-build-release.yaml

```diff
@@ -56,6 +49,12 @@ jobs:
           prerelease: false
           automatic_release_tag: "${{ steps.version.outputs.version }}"
 
+      - name: Build package
+        shell: bash
+        run: |
+          set -euo pipefail
+          python3 setup.py build sdist
+
```

Comment:
Instead of moving this below the release action, we could do something like this:

```
set -euo pipefail
git tag "${{ steps.version.outputs.version }}"
python3 setup.py build sdist
git tag -d "${{ steps.version.outputs.version }}"
```

This would avoid creating a version in the case of a failed build